docs/guides/databases/mariadb/backup-mariadb-mysql-to-object-storage-with-restic/index.md

```diff
@@ -1,16 +1,18 @@
 ---
+slug: backup-mariadb-mysql-to-object-storage-with-restic
 author:
   name: Andy Heathershaw
   email: andy@andysh.uk
-description: 'This guide shows you how to use Restic to backup your MariaDB or MySQL databases onto Linode Object Storage.'
-og_description: 'Learn how to backup your MariaDB and MySQL databases off your Linode and onto Linode Object Storage with Restic.'
+description: 'Restic is a backup utility written in Go. This guide shows how to configure Restic to backup your MariaDB (or MySQL) databases onto Linode Object Storage.'
+og_description: 'Restic is a backup utility written in Go. This guide shows how to configure Restic to backup your MariaDB (or MySQL) databases onto Linode Object Storage.'
 keywords: ['mariadb','mysql','backup','backups','restic','off-site backups','Object Storage']
+tags: ['mariadb', 'mysql', 'automation']
```

Comment:
We're using tags as filters for the search experience in the new docs UI, so I added some here. These tags come from a standard pool of tags that we're maintaining internally, so you don't have visibility over them, and so you don't really need to worry about adding them yourself